tensorpack/dataflow/imgaug/crop.py

```diff
@@ -127,8 +137,10 @@ def _get_augment_params(self, img):
     def _augment(self, img, newbox):
         return newbox.roi(img)
 
-    def _augment_coords(self, coords, param):
-        raise NotImplementedError()
+    def _augment_coords(self, coords, newbox):
```

Comment:
Don't have to worry too much about these weird augmentors.. They are not quite well defined and not used by anyone, I think.